projects/sandbox/datagen/datagen/utils/timeslides.py

```diff
@@ -24,10 +24,13 @@ def __init__(
         waveform_duration: float,
         max_shift: float,
         jitter: float,
-        parameter_file: Optional[Path] = None,
+        buffer: float = 0,
+        spacing: float = 0,
     ) -> None:
         self.prior = prior
```

Comment:
Would it make sense to call `prior()` here rather than constructing the dict each time?

Reply:
(As in constructing it by calling `prior()` in each `__call__`)

Reply:
Yes, agreed.